tools/exe/palletjack2ipahost

```diff
@@ -45,6 +45,15 @@ Synchronize the warehouse with the IPA server."
     }
 
     required_option :ipa
+
+    @option_checks << Proc.new do
+      if options[:reinstall] then
+        unless options[:system] then
+          raise ArgumentError
+        end
+      end
+    end
```

Comment:
Add `required_option :system` inside the `opts.on` block for `--reinstall` instead of reimplementing `required_option` here.